app/styles/modules/message.scss

```diff
@@ -91,11 +100,28 @@ article.message[data-status="typing"] > div {
           line-height: 28px;
         }
       .message-status {
-        font-size: 14px;
+        display: inline-block;
+        vertical-align: top;
+        width: 115px;
+        font-size: 13px;
         line-height: 24px;
+        text-align: right;
         color: #999;
+      }
+      .message-geo, .info_indent {
```

Comment:
https://github.com/QuickBlox/CodeStyle-Guidelines-for-Developers/tree/master/web/css
3. When grouping selectors, keep individual selectors to a single line;